jacodb-core/src/main/kotlin/org/jacodb/impl/fs/ByteCodeConverter.kt

```diff
@@ -124,6 +124,17 @@ private fun MethodNode.asMethodInfo(): MethodInfo {
     )
 }
 
+private fun MethodNode.argumentName(argIndex :Int): String? {
+    localVariables?.let {
+        (argIndex + 1 - (access and Opcodes.ACC_STATIC).countOneBits()).run {
+            if (it.size > this) {
+                return ArrayList(it).sortedBy(LocalVariableNode::index)[this].name
+            }
+        }
+    }
+    return null
```

Comment:
Seems like fallback `parameters?.get(index)?.name` is missing. Not sure that  localVariables are mandatory for runtime.

Reply:
Thanks, fallback is here now